package/icu/icu.mk

```diff
@@ -17,6 +17,20 @@ ICU_SITE = http://download.icu-project.org/files/icu4c/$(ICU_VERSION)
 ICU_LICENSE = ICU License
 ICU_LICENSE_FILES = license.html
 
+ifeq ($(ICU_VERSION),58.2)
+define ICU_USE_CUSTOM_ICUDATA
+    rm -f $(@D)/source/data/in/icudt58l.dat
+    cp package/icu/icudt58l.dat $(@D)/source/data/in/
+endef
+define HOST_ICU_USE_CUSTOM_ICUDATA
+    rm -f $(HOST_ICU_DIR)/source/data/in/icudt58l.dat
+    cp package/icu/icudt58l.dat $(HOST_ICU_DIR)/source/data/in/
+endef
+endif
+
+ICU_POST_PATCH_HOOKS += ICU_USE_CUSTOM_ICUDATA
+HOST_ICU_POST_PATCH_HOOKS += HOST_ICU_USE_CUSTOM_ICUDATA
+
```

Comment:
Same here, dupe of https://github.com/WebPlatformForEmbedded/buildroot/pull/79 ?

Reply:
Added some optimisation, will check in master and update